src/vcpkg/build.cpp

```diff
@@ -1574,6 +1576,14 @@ namespace vcpkg::Build
                                                   variable_value);
                     }
                     break;
+                case VcpkgTripletVar::NO_COMPILER_TRACKING:
+                    if (Strings::case_insensitive_ascii_equals(variable_value, "1") ||
+                        Strings::case_insensitive_ascii_equals(variable_value, "on") ||
+                        Strings::case_insensitive_ascii_equals(variable_value, "true"))
```

Comment:
I think this should be factored out into a function. I see the code is duplicated above.

Reply:
You are right and that's a good idea. That being said, I tend to try to keep my pull requests super focus when I collaborate on open source projects. This way they can be reviewed and integrated more easily. With that in mind, I think we should factorize this code in a function in another pull request.